src/OpenIddict.Core/OpenIddictBuilder.cs

```diff
@@ -1,19 +1,404 @@
+        public virtual OpenIddictBuilder SetAuthorizationEndpointPath(PathString path) {
+            return Configure(options => options.AuthorizationEndpointPath = path);
+        }
+
+        /// <summary>
+        /// Sets the relative path corresponding to the logout endpoint.
+        /// </summary>
+        /// <param name="path">The relative path of the logout endpoint.</param>
+        /// <returns>The <see cref="OpenIddictBuilder"/>.</returns>
+        public virtual OpenIddictBuilder SetLogoutEndpointPath(PathString path) {
+            return Configure(options => options.LogoutEndpointPath = path);
+        }
+
+        /// <summary>
+        /// Sets the relative path corresponding to the token endpoint.
+        /// </summary>
+        /// <param name="path">The relative path of the token endpoint.</param>
+        /// <returns>The <see cref="OpenIddictBuilder"/>.</returns>
+        public virtual OpenIddictBuilder SetTokenEndpointPath(PathString path) {
+            return Configure(options => options.TokenEndpointPath = path);
+        }
+
+        /// <summary>
+        /// Sets the access token lifetime, after which client applications must retrieve
+        /// a new access token by making a grant_type=refresh_token token request
+        /// or a prompt=none authorization request, depending on the selected flow.
+        /// Using long-lived access tokens or tokens that never expire is not recommended.
+        /// </summary>
+        /// <param name="lifetime">The access token lifetime.</param>
+        /// <returns>The <see cref="OpenIddictBuilder"/>.</returns>
+        public virtual OpenIddictBuilder SetAccessTokenLifetime(TimeSpan lifetime) {
+            return Configure(options => options.AccessTokenLifetime = lifetime);
+        }
+
+        /// <summary>
+        /// Sets the refresh token lifetime, after which client applications must get
+        /// a new authorization from the user. When sliding expiration is enabled,
+        /// a new refresh token is always issued to the client application,
+        /// which prolongs the validity period of the refresh token.
         /// </summary>
-        public ICollection<OpenIddictModule> Modules { get; } = new List<OpenIddictModule>();
+        /// <param name="lifetime">The refresh token lifetime.</param>
+        /// <returns>The <see cref="OpenIddictBuilder"/>.</returns>
+        public virtual OpenIddictBuilder SetRefreshTokenLifetime(TimeSpan lifetime) {
+            return Configure(options => options.RefreshTokenLifetime = lifetime);
+        }
 
         /// <summary>
-        /// Gets or sets the options used by OpenIddict.
+        /// Sets JWT as the default token format for access tokens.
         /// </summary>
-        public OpenIddictOptions Options { get; set; } = new OpenIddictOptions();
+        /// <returns>The <see cref="OpenIddictBuilder"/>.</returns>
+        public virtual OpenIddictBuilder UseJsonWebTokens() {
```

Comment:
Actually, `JWT` means `JSON Web Token`, so `JwtTokens` literally means `JSON Web Token Tokens` :smile: 
I'm considering renaming it in ASOS.